MAINTAINERS

```diff
@@ -0,0 +1,6 @@
+# Repo-Level Maintainers (in alphabetical order)
+# Note: This is for the notaryproject/notation-hashicorp-vault repo
+Alexander Scheel <alex.scheel@hashicorp.com> (@cipherboy)
+Bingqi Shang <bingqi.shang.apply@gmail.com> (@OliverShang)
+Patrick Zheng <patrickzheng@microsoft.com> (@patrickzheng200)
```

Comment:
This user doesn't match the CODEOWNERS entry. Also the new handle is @Two-Hearts